locale/in/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,2795 @@
+" chosen prefix `dt`, the following are all valid module names:"
+msgstr ""
+"कस्टम उपसर्ग के साथ मॉड्यूल नाम के बाद एक ``एकल अंडरस्कोर`` `_` होता है, जो इस विकल्प "
+"को मौजूदा पैकेज के साथ अधिक लचीला और पिछड़ा संगत बनाता है। जब एक कस्टम मॉड्यूल उपसर्ग "
+"निर्दिष्ट किया जाता है, तो डिफ़ॉल्ट अभी भी उपलब्ध होता है। उदाहरण के लिए चयनित उपसर्ग `dt` के "
+"साथ `दिनांक-समय` नामक एक पैकेज को ध्यान में रखते हुए, निम्नलिखित सभी मान्य मॉड्यूल नाम हैं:"
+
+#: ../../pages/spec/naming.md:116
+msgid "Package names"
+msgstr "पैकेज के नाम"
+
+#: ../../pages/spec/naming.md:118
+msgid ""
+"All packages in FPM registries must have unique names, hence they must abide "
+"to the following rules"
+msgstr ""
+"एफपीएम रजिस्ट्रियों में सभी पैकेजों के अद्वितीय नाम होने चाहिए, इसलिए उन्हें निम्नलिखित नियमों का पालन करना चाहिए|"
+
+#: ../../pages/spec/naming.md:120
+msgid "All package names shall be valid Fortran names;"
+msgstr "सभी पैकेज नाम वैध फोरट्रान नाम होंगे;"
+
+#: ../../pages/spec/naming.md:121
+msgid "Dash characters (`-`) are also allowed, and are treated by fpm as underscores;"
+msgstr ""
+"डैश वर्ण (`-`) की भी अनुमति है, और fpm द्वारा अंडरस्कोर के रूप में व्यवहार किया जाता है;"
+
+#: ../../pages/spec/naming.md:122
+msgid ""
+"Package names may contain uppercase and lowercase characters, but their "
+"unique identification is made case insensitive;"
+msgstr ""
+"पैकेज के नाम में अपरकेस और लोअरकेस अक्षर हो सकते हैं, लेकिन उनकी विशिष्ट पहचान "
+"को केस असंवेदनशील बना दिया जाता है;"
+
+#: ../../pages/spec/naming.md:123
+msgid "No duplicate package names are allowed within the same namespace."
+msgstr "एक ही नामस्थान में डुप्लीकेट पैकेज नामों की अनुमति नहीं है।"
+
+#: ../../pages/spec/naming.md:125
+msgid "*Examples of valid package names:*"
+msgstr "*वैध पैकेज नामों के उदाहरण:*""
+
+#: ../../pages/spec/naming.md:134
+msgid "*Examples of invalid package names:*"
+msgstr "*अवैध पैकेज नामों के उदाहरण:*"
+
+#: ../../pages/spec/naming.md:145
+msgid "Manifest Settings"
+msgstr "प्रकट सेटिंग्स"
+
+#: ../../pages/spec/naming.md:148
+msgid "Key facts:"
+msgstr "महत्वपूर्ण तथ्यों:"
+
+#: ../../pages/spec/naming.md:149
+msgid ""
+"FPM does not apply naming requirements by default. If you want them, enable "
+"them in `fpm.toml`"
+msgstr ""
+"एफपीएम डिफ़ॉल्ट रूप से नामकरण की आवश्यकताओं को लागू नहीं करता है। यदि आप उन्हें चाहते हैं, "
+"तो उन्हें `fpm.toml` में सक्षम करें"
+
+#: ../../pages/spec/naming.md:150
+msgid "FPM registries mandatorily require them. Ensure `fpm.toml` enables them."
+msgstr ""
+"एफपीएम रजिस्ट्रियों को अनिवार्य रूप से उनकी आवश्यकता होती है। सुनिश्चित करें कि `fpm.toml` "
+"उन्हें सक्षम करता है।"
+
+#: ../../pages/spec/naming.md:151
+msgid ""
+"Enable standard prefix with `module-naming=true`, custom prefix with `module-"
+"naming=\"prefixname\"`."
+msgstr ""
+"`मॉड्यूल-नामकरण = सत्य` के साथ मानक उपसर्ग सक्षम करें, "
+"`मॉड्यूल-नामकरण = \"उपसर्गनाम \"` के साथ कस्टम उपसर्ग।"
```

Comment:
I think this is problematic. The actual text that you have to put into `fpm.toml` has to be in English, `module-naming=true`. So I would not translate those technical terms into Hindi, but keep them in English.

Reply:
chatGPT suggests the following instead:
```
`module-naming=true` के साथ मानक प्रिफ़िक्स सक्षम करें, `module-naming=\"prefixname\"` के साथ कस्टम प्रिफ़िक्स सक्षम करें।
```
I don't know if it is a better translation, but it recognizes the lines of code and does not translate them, which I think is better.

Reply:
@certik Actually that's a better translation. Apparently different translation tools do give different outputs but no worries since you have pointed this out, I'l go through the whole code once again and make sure that any technical terms stays in the English language only